### ISSUE TITLE
About BatchNorm

### ISSUE THREAD

**Issue by mazzzystar** [r9y9/tacotron_pytorch#19]
https://github.com/r9y9/tacotron_pytorch/blob/5f41d9d8f70a299a49f02aa5422478e1693ebe93/tacotron_pytorch/tacotron.py#L36

In your implementation code, it says "following tensorflow's default parameters".  However, the `momentum` in PyTorch is the opposite with Tensroflow.
```python
# PyTorch
x(n+1) = (1 - momentum) * avg(x(1-n)) + momentum * x(n)  # so default momentum=0.1
# Tensorflow
x(n+1) = momentum* avg(x(1-n)) + (1-momentum) * x(n)   # so default momentum=0.99
```

What's your idea on that ?   And I consider why using so large `eps`(1e-3) for audio ? The default `eps` for PyTorch is 1e-5

**Comment by r9y9** [comment]
I didn't look into the tensorflow implementation carefully, so that's simply my mistake. Same reason for `eps` as well.

**Comment by r9y9** [comment]
Thanks again for pointing out the issue. Fixed.
